src/main/resources/application-test.yml

```
@@ -0,0 +1,29 @@
+    com.devs.roamance: info
+
+application:
+  frontend:
+    url: http://localhost:3000
+  security:
+    jwt:
+      secret: testsecret
+      access-token-expiration: 3600000
+      refresh-token-expiration: 86400000
+  gemini:
+    api-key: sdlkfnsdlk
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)